### DISCUSSION TITLE
Configure "Select left" or "Select right" as context menu option

### DISCUSSION THREAD

**Discussion by twistah03** [WinMerge/winmerge#1297] [Ideas]
It's possible to add in option to configure "Select left" or "Select right" as first choice in context menu?

**Comment by sdottaka** [comment]
To display "Select Left" in the Explorer context menu, check the "Enable advanced menu" checkbox in the Shell Integration category of the Options dialog.

![image](https://user-images.githubusercontent.com/98126/164339606-14e1e2a7-c3a1-40a3-af4e-7590e7f1d918.png)

**Comment by twistah03** [comment]
Yes. Thank you for tip.
But in context menu appears these options:
![image](https://user-images.githubusercontent.com/104089154/164375634-2d0fb64f-e9db-43c8-a411-e46ac3e9467b.png)

My idea is to have this option configurable (in Enable advance menu), to be able to "Select Right" as first option.
